source/_integrations/caldav.markdown

```diff
@@ -133,6 +133,10 @@ verify_ssl:
   required: false
   type: boolean
   default: true
+ssl_client_cert:
+  description: Used to enable SSL client authentication. Value points to a file on the filesystem containing both the unencrypted private key and certificate for the client. This file should be protected. Make sure the owner is set to the user Home Assistant runs as and permissions are set to be readable by that user only. You should consider putting the file outside of the Home Assistant configuration directories as well. 
+  required: optional
+  type: string
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1:

Reply:
@frenck , That's an interesting decision. I personally don't own the caldav integration. Even though I'm spending time on it getting it to work for my needs I also don't want to be responsible for it. If someone wants to port caldav over to use the new config flow model, that would be helpful and I can work with them on adding this new feature.

I do have security concerns around storing and managing a private key via the UI. A private key defines an identity and needs to be kept as protected as possible. To use the new model implies the UI loading and managing of the key which is a big risk as the UI may contain unforeseen bugs and issues.

Reply:
>  I personally don't own the caldav integration. Even though I'm spending time on it getting it to work for my needs I also don't want to be responsible for it. If someone wants to port caldav over to use the new config flow model, that would be helpful and I can work with them on adding this new feature

If you are not going to implement the required changes, I would suggest on closing the PRs.